Neos.ContentRepository.LegacyNodeMigration/Classes/NodeDataToEventsProcessor.php

```diff
@@ -316,7 +316,12 @@ public function extractPropertyValuesAndReferences(array $nodeDataRow, NodeType
         }
 
         foreach ($decodedProperties as $propertyName => $propertyValue) {
-            $type = $nodeType->getPropertyType($propertyName);
+            try {
+                $type = $nodeType->getPropertyType($propertyName);
+            } catch (\InvalidArgumentException $exception) {
```

Comment:
same here

Reply:
Thanks for the feedback @bwaidelich, that comes from https://github.com/neos/neos-development-collection/blob/9.0/Neos.ContentRepository.Core/Classes/NodeType/NodeType.php in line 422.

Reply:
Ah of course, I looked at the wrong branch 👀 
Thanks for the clarification!